<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMOTedarikciListeWidget</class>
 <widget class="QWidget" name="TMOTedarikciListeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpArama">
     <property name="title">
      <string>Arama</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="grpTedarikciAdi">
        <property name="title">
         <string>Tedarikçi Adı</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="lblTedarikciAdiDegeri">
             <property name="text">
              <string>Değer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leTedarikciAdiDegeri"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="grpTedarikciAdiFiltreTuru">
           <property name="title">
            <string>Filtre Türü</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QRadioButton" name="rbTedarikciAdiIleBaslayan">
              <property name="text">
               <string>İle Başlayan</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbTedarikciAdiIleBiten">
              <property name="text">
               <string>İle Biten</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbTedarikciAdiIceren">
              <property name="text">
               <string>İçeren</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grpYetkiliKisi">
        <property name="title">
         <string>Yetkili Kişi</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="lblYetkiliKisiDegeri">
             <property name="text">
              <string>Değer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leYetkiliKisiDegeri"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="grpYetkiliKisiFiltreTuru">
           <property name="title">
            <string>Filtre Türü</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QRadioButton" name="rbYetkiliKisiIleBaslayan">
              <property name="text">
               <string>İle Başlayan</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbYetkiliKisiIleBiten">
              <property name="text">
               <string>İle Biten</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbYetkiliKisiIceren">
              <property name="text">
               <string>İçeren</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnAra">
          <property name="text">
           <string>Ara</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tblVeriler"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rbTedarikciAdiIleBaslayan</sender>
   <signal>clicked()</signal>
   <receiver>TMOTedarikciListeWidget</receiver>
   <slot>arama_yap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbTedarikciAdiIleBiten</sender>
   <signal>clicked()</signal>
   <receiver>TMOTedarikciListeWidget</receiver>
   <slot>arama_yap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbTedarikciAdiIceren</sender>
   <signal>clicked()</signal>
   <receiver>TMOTedarikciListeWidget</receiver>
   <slot>arama_yap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>434</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbYetkiliKisiIleBaslayan</sender>
   <signal>clicked()</signal>
   <receiver>TMOTedarikciListeWidget</receiver>
   <slot>arama_yap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>675</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>1262</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbYetkiliKisiIleBiten</sender>
   <signal>clicked()</signal>
   <receiver>TMOTedarikciListeWidget</receiver>
   <slot>arama_yap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>854</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>1307</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbYetkiliKisiIceren</sender>
   <signal>clicked()</signal>
   <receiver>TMOTedarikciListeWidget</receiver>
   <slot>arama_yap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1019</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>1305</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leYetkiliKisiDegeri</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TMOTedarikciListeWidget</receiver>
   <slot>arama_yap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>732</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>1217</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leTedarikciAdiDegeri</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TMOTedarikciListeWidget</receiver>
   <slot>arama_yap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>arama_yap()</slot>
 </slots>
</ui>
